tap_adwords/__init__.py

```diff
@@ -877,8 +877,21 @@ def check_selected_fields(stream, field_list, sdk_client):
                 field.xmlAttributeName, ",".join(field_errors)))
 
     if errors:
-        raise Exception("Fields selection violates Google's exclusion rules:\n\t{}" \
-                        .format("\n\t".join(errors)))
+        if sdk_client.user_agent == "Stitch Tap (+support@stitchdata.com)":
```

Comment:
Can this be more generic? I'd prefer just straight lowercase `stitch` in user_agent to differentiate, since it's informational, but "Stitch Tap" or something works too. I'd just rather not have this code die or have to go back to change it if the user Agent changes.

The conditional also doesn't really care about the user agent's exact match, it cares who is running the tap to provide better info, so just `"Stitch" in user_agent` makes the most sense to me.

Reply:
Given that that `user_agent` is hard coded into orchestrator I think it makes sense to rely on it being constant everywhere. Happy to discuss IDL but I think an exact match should be expected here since we're no more likely to match case insensitive stitch if the user agent changes than we are to match this.

Reply:
My thought is that the email is less important of an identifier, and might actually change sooner than later. Just removing the email would make me feel better. I just don't want to come back and fix this, unless it's ab big change.